corehq/apps/hqmedia/models.py

```diff
@@ -606,15 +606,16 @@ def get_media_ref_kwargs(self, module, module_index, form=None,
             'is_menu_media': is_menu_media,
         }
 
-    def remove_unused_mappings(self):
+    def remove_unused_mappings(self, additional_permitted_paths=[]):
```

Comment:
generally it's advisable not to use lists and dicts as default arguments, since they are mutable and scoped to where the method is defined. should change to a tuple or `None`. 

http://effbot.org/zone/default-values.htm